projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -454,6 +457,35 @@ export class RecordSearchComponent implements OnInit, OnChanges, OnDestroy {
     return null;
   }
 
+  /**
+   * Export formats.
```

Comment:
This documentation seems a bit useless, can you mention at least that this values comes from the configuration?

Reply:
Ok done.